src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointOptions.tsx

```diff
@@ -17,25 +18,37 @@ function BreakpointLineContents({ breakpoint }: BreakpointProps) {
   const replayClient = useContext(ReplayClientContext);
   const { sourceId } = breakpoint.location;
 
-  const sourceLines = getSourceLinesSuspense(sourceId, replayClient);
+  const streamingSourceContent = getStreamingSourceContentsSuspense(replayClient, sourceId);
+  const parsed = getParsedValueIfCached(streamingSourceContent, null)?.value;
 
-  const snippet = useMemo(() => {
+  const parsedTokensByLine = parsed?.parsedTokensByLine;
+  const rawTextByLine = parsed?.rawTextByLine;
+
+  const { snippet, tokens } = useMemo((): { snippet: string; tokens: ParsedToken[] } => {
     const { column, line } = breakpoint.location;
     let snippet = "";
+    let tokens: ParsedToken[] = [];
+
+    if (rawTextByLine && rawTextByLine.length > 0) {
```

Comment:
If `rawTextByLine` has _some_ lines but not as many as the specified index (`line - 1`) this check will cause a runtime error below– because we'll end up calling `.slice` on undefined.